ios/xcode/FluentUITests.xcconfig

```diff
@@ -0,0 +1,14 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES
+CODE_SIGN_STYLE = Automatic
+DEVELOPMENT_TEAM = UBF8T346G9
```

Comment:
Do we need this?

Reply:
We don't necessarily need it to run the tests. But it'll avoid the warnings in Xcode:

![Screen Shot 2020-11-19 at 8 48 03 PM](https://user-images.githubusercontent.com/68076145/99760476-a9687680-2aa8-11eb-9cb9-cc293cf4027b.png)

Reply:
i don't think i like developement team tag in public?

Reply:
also that is specific to office team afaik

Reply:
This was already public in the project file. These are just being moved to xcconfig files.

That is the Apple Team ID for Microsoft.
It's part of a lot of App Bundles distributed publicly by Microsoft.
An example is Teams App which has this in the embedded.provisionprofile file inside its application Bundle.

```
	<key>TeamIdentifier</key>
	<array>
		<string>UBF8T346G9</string>
	</array>
```